src/DataTable/DropdownFilters.jsx

```diff
@@ -25,18 +25,22 @@ const DropdownFilters = () => {
 
   return (
     <div className="pgn__data-table-filters">
-      {breakoutFilters.length > 0 && breakoutFilters.map((column) => (<div className="mr-2" key={column.Header}>{column.render('Filter')}</div>))}
+      {breakoutFilters.length > 0 && breakoutFilters.map((column) => (
+        <div className="pgn__data-table-filters-breakout-filter" key={column.Header}>
```

Comment:
Updating the formatting and removed `mr-2` in favor of `pgn__data-table-filters-breakout-filter`